indico/web/client/js/utils/sort.js

```diff
@@ -0,0 +1,29 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2022 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+// Same as natural_sort_key
+export function natSortCompare(a, b) {
```

Comment:
This looks so much cleaner than this strnatcmp i pasted from phpjs a loooong time ago :D